tests/msc3930_test.go

```diff
@@ -0,0 +1,182 @@
+//go:build msc3930
+// +build msc3930
+
+// This file contains tests for "push rules of polls" as defined by MSC3930.
+// The MSC that defines the design of the polls system is MSC3381.
+//
+// Note that implementation of MSC3381 is not required by the homeserver to
+// pass the tests in this file.
+//
+// You can read either MSC using at the links below.
+// Polls: https://github.com/matrix-org/matrix-doc/pull/3381
+// Push rules for polls: https://github.com/matrix-org/matrix-doc/pull/3930
+
+package tests
+
+import (
+	"math"
+	"testing"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+const pollResponsePushRuleID = ".org.matrix.msc3930.rule.poll_response"
```

Comment:
nit: To make this name consistent with the others, we might want to omit the `Push`.
```suggestion
const pollResponseRuleID = ".org.matrix.msc3930.rule.poll_response"
```